src/test/java/ibm/jceplus/junit/openjceplusfips/multithread/TestHmacSHA3_224.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright IBM Corp. 2023
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+
+package ibm.jceplus.junit.openjceplusfips.multithread;
+
+
+import ibm.jceplus.junit.base.BaseTestHmacSHA3_224;
+import ibm.jceplus.junit.openjceplus.Utils;
+
+public class TestHmacSHA3_224 extends ibm.jceplus.junit.base.BaseTestHmacSHA3_224 {
+
+    //--------------------------------------------------------------------------
+    //
+    //
+    static {
+        Utils.loadProviderTestSuite();
+    }
+
+    //--------------------------------------------------------------------------
+    //
+    //
+    public TestHmacSHA3_224() {
+        super(Utils.TEST_SUITE_PROVIDER_NAME);
+    }
+
+    public void testHmacSHA3_224() throws Exception {
+        System.out.println("executing testHmacSHA3_224");
+        BaseTestHmacSHA3_224 bt = new BaseTestHmacSHA3_224(providerName);
+        bt.run();
+
+    }
+
+    public static void main(String[] args) {
+        String[] nargs = {
+                ibm.jceplus.junit.openjceplusfips.multithread.TestHmacSHA3_224.class.getName()};
+        junit.textui.TestRunner.main(nargs);
+    }
```

Comment:
For new test i think we can remove main method of the test as we can run individual tests through surefire and mvn.

Reply:
Removed.